apps/communications/contacts/js/views/details.js

```diff
@@ -200,7 +200,7 @@ contacts.Details = (function() {
     cover.addEventListener('touchstart', onTouchStart, true);
   };
 
-  var render = function cd_render(currentContact, fbContactData) {
+  var render = function cd_render(currentContact, fbContactData, staticMode) {
```

Comment:
please document what does it mean 'staticMode' . What does it mean? can we have another more meaningful name?